quark/utils/graph.py

```diff
@@ -191,30 +192,37 @@ def call_graph(call_graph_analysis, output_format="png"):
         )
 
     if second_call != second_api:
-        dot.edge(
-            f"{parent_function.full_name}",
-            f"{second_wrapper[-1].full_name}",
-            "Second Call",
-            fontname="Courier New",
-        )
+        try:
+            dot.edge(
+                f"{parent_function.full_name}",
+                f"{second_wrapper[-1].full_name}",
+                "Second Call",
+                fontname="Courier New",
+            )
 
-        dot.edge(
-            f"{second_wrapper[0].full_name}",
-            f"{second_api.full_name}",
-            "calls",
-            fontname="Courier New",
-        )
+            dot.edge(
+                f"{second_wrapper[0].full_name}",
+                f"{second_api.full_name}",
+                "calls",
+                fontname="Courier New",
+            )
+        except IndexError:
+            pass
     else:
-        dot.edge(
-            f"{parent_function.full_name}",
-            f"{second_api.full_name}",
-            "Second Call",
-            fontname="Courier New",
-        )
+        try:
+            dot.edge(
+                f"{parent_function.full_name}",
+                f"{second_api.full_name}",
+                "Second Call",
+                fontname="Courier New",
+            )
+        except IndexError:
+            pass
 
-    dot.render(
-        f"call_graph_image/{parent_function.name}_{first_call.name}_{second_call.name}"
-    )
+    fname = f"{parent_function.name}_{first_call.name}_{second_call.name}"
+    fname = sanitize_filename(fname, replacement_text="_")
+    fname = os.path.join("call_graph_image", fname)
+    dot.render(fname)
 
```

Comment:
I suggest naming the variable to `output_path` to improve code readability. What do you think?

Reply:
Done.

Reply:
```
    fname = f"{parent_function.name}_{first_call.name}_{second_call.name}"
    fname = sanitize_filename(fname, replacement_text="_")
    output_path = os.path.join("call_graph_image", fname)
```

It seems ok. or any suggestion ?